Podcasts/EpisodeView/EpisodePresenter.swift

```diff
@@ -0,0 +1,38 @@
+//  Created by Maxos on 5/3/23.
+
+import Foundation
+
+protocol EpisodeView: AnyObject {
+    func display(_ episode: [Episode])
+}
+
+final class EpisodePresenter {
+    weak var view: EpisodeView?
+    private var episodeID: String
+    
+    init(view: EpisodeView?, episodeID: String) {
```

Comment:
It is better to define required parameters first, optional - second like
`init(episodeID: String, view: EpisodeView?)`